docs/source/developingplugins.rst

```diff
@@ -41,6 +41,8 @@ and is meant to categorize rules; you could use the
 letter 'S' to denote rules that enforce security checks
 for example.
 
+An important thing to note when running custom implemented rules. Run ``pip install - e`` inside the plugin folder so  custom rules in linting are included.
```

Comment:
```suggestion
An important thing to note when running custom implemented rules:  Run ``pip install - e .`` inside
the plugin folder so custom rules in linting are included.
```

Reply:
Above, I suggested splitting the long line. I also added `.` to the `pip install` command -- I think that's required.

Reply:
Splitting the line seems fine, however I don't think the `.` was necessary, maybe a `,` outside of the code block? It just may confuse users with actually adding the period to the command, when in reality we don't want to add that period to the command. Good suggestion on the line split!

Reply:
I'm pretty sure it is needed, and also that you can't have a space between the dash and the `e`. Will make a suggestion on the latest PR.